packages/react-app/package.json

```diff
@@ -22,7 +22,9 @@
     "type-check": "tsc --pretty --noEmit --skipLibCheck --esModuleInterop --strict --jsx preserve",
```

Comment:
package.json inside the react app, nothing exciting here, some type files and some commands